projects/annotation-ui-lib/src/lib/document-viewer/document-viewer.component.ts

```diff
@@ -52,11 +52,17 @@ export class DocumentViewerComponent implements OnChanges, OnInit {
             if (metadata && metadata._links) {
               const url = this.urlFixer.fixDm(metadata._links.binary.href, this.baseUrl);
               const dmDocumentId = this.viewerFactoryService.getDocumentId(metadata);
-              this.annotationStoreService.getAnnotationSet(this.baseUrl, dmDocumentId).subscribe(annotationSet => {
+              if (this.annotate) {
+                this.annotationStoreService.getAnnotationSet(this.baseUrl, dmDocumentId).subscribe(annotationSet => {
+                  this.viewerComponent =
+                    this.viewerFactoryService.buildComponent(this.viewerAnchor.viewContainerRef,
+                      metadata.mimeType, url, this.baseUrl, metadata._links.self.href, this.annotate, annotationSet.body);
+                });
+              } else {
                 this.viewerComponent =
                   this.viewerFactoryService.buildComponent(this.viewerAnchor.viewContainerRef,
-                    metadata.mimeType, url, this.baseUrl, metadata._links.self.href, this.annotate, annotationSet.body);
-              });
+                    metadata.mimeType, url, this.baseUrl, metadata._links.self.href, this.annotate, null);
```

Comment:
build component is repeated twice, maybe have that outside the if condition.

Reply:
I moved it to a separate func